content/ar/docs/contribute/localization.md

```diff
@@ -24,4 +24,30 @@ content_type: concept
 
 English | العربية
```

Comment:
Tip: write these sorted in alphabetical order of the English term; that can make them easier to find.

Reply:
done